Problem: 'sudo make devinstall' fails on Maxim's VM

While following [README_developers.md](https://github.com/Seagate/hare/blob/829418b8d6606d58704a1c63408992c5907724fd/README_developers.md),
@max-seagate.medved got error of `sudo make devinstall`:
```
$ sudo make devinstall
[...]
--> Building hax .whl package
  File "setup.py", line 16
    user_options: List[Tuple[str, str, str]] = []
                ^
SyntaxError: invalid syntax
make: *** [hax/dist/hax-0.1.0-cp36-cp36m-linux_x86_64.whl] Error 1
```

Git commit: 829418b8d6606d58704a1c63408992c5907724fd

```
> python3.6 --version
Python 3.6.3
> dhall --version
1.27.0
> consul --version
Consul v1.6.1
Protocol 2 spoken by default, understands 2 to 3 (agent will automatically use protocol >2 when speaking to compatible agents)
```

<details>
<summary>See full output of <code>sudo make devinstall</code></summary>

```
$ sudo make devinstall
--> Installing hax directories
install: creating directory ‘/var/mero/hax’
--> Installing directories
install: creating directory ‘/var/lib/hare’
install: creating directory ‘/opt/seagate’
install: creating directory ‘/opt/seagate/hare’
install: creating directory ‘/opt/seagate/hare/libexec’
install: creating directory ‘/opt/seagate/hare/libexec/hare’
--> Installing cfgen
install: creating directory ‘/opt/seagate/hare/bin’
‘/opt/seagate/hare/bin/cfgen’ -> ‘/work/hare/cfgen/cfgen’
--> Initializing virtual env in /work/hare/.py3venv
--> Installing pip modules in virtual env
    processing cfgen/requirements.txt
Collecting PyYAML (from -r cfgen/requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/e3/e8/b3212641ee2718d556df0f23f78de8303f068fe29cdaa7a91018849582fe/PyYAML-5.1.2.tar.gz (265kB)
    100% |████████████████████████████████| 266kB 945kB/s 
Collecting ply (from -r cfgen/requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/a3/58/35da89ee790598a0700ea49b2a66594140f44dec458c07e8e3d4979137fc/ply-3.11-py2.py3-none-any.whl (49kB)
    100% |████████████████████████████████| 51kB 11.1MB/s 
Installing collected packages: PyYAML, ply
  Running setup.py install for PyYAML ... done
Successfully installed PyYAML-5.1.2 ply-3.11
You are using pip version 9.0.1, however version 19.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.                                                                                                                                                                                                      
    processing hax/requirements.txt
Collecting flake8 (from -r hax/requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/f8/1f/7ea40d1e4146ea55dbab41cda1376db092a75794914169aabd7e8d7a7def/flake8-3.7.9-py2.py3-none-any.whl (69kB)
    100% |████████████████████████████████| 71kB 670kB/s 
Collecting mypy (from -r hax/requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/8a/ba/c458622f83d68f6ce9f6e8fe3f9910f0244cd274a0e6dee85f37c985a150/mypy-0.720-py3-none-any.whl (1.6MB)
    100% |████████████████████████████████| 1.6MB 828kB/s 
Collecting pkgconfig (from -r hax/requirements.txt (line 3))
  Downloading https://files.pythonhosted.org/packages/b4/2c/bf434cb5a6590417e1d4468050ec317ea17fd6231c2a256df4646c11e588/pkgconfig-1.5.1-py2.py3-none-any.whl
Collecting python-consul (from -r hax/requirements.txt (line 4))
  Downloading https://files.pythonhosted.org/packages/3f/d0/59bc5f1c6c4d4b498c41d8ce7052ee9e9d68be19e16038a55252018a6c4d/python_consul-1.1.0-py2.py3-none-any.whl
Requirement already satisfied: setuptools in ./.py3venv/lib/python3.4/site-packages (from -r hax/requirements.txt (line 5))
Collecting wheel (from -r hax/requirements.txt (line 6))
  Downloading https://files.pythonhosted.org/packages/00/83/b4a77d044e78ad1a45610eb88f745be2fd2c6d658f9798a15e384b7d57c9/wheel-0.33.6-py2.py3-none-any.whl
Collecting typing; python_version < "3.5" (from flake8->-r hax/requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/fe/2e/b480ee1b75e6d17d2993738670e75c1feeb9ff7f64452153cf018051cc92/typing-3.7.4.1-py3-none-any.whl
Collecting entrypoints<0.4.0,>=0.3.0 (from flake8->-r hax/requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/ac/c6/44694103f8c221443ee6b0041f69e2740d89a25641e62fb4f2ee568f2f9c/entrypoints-0.3-py2.py3-none-any.whl
Collecting pyflakes<2.2.0,>=2.1.0 (from flake8->-r hax/requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/84/f2/ed0ffb887f8138a8fe5a621b8c0bb9598bfb3989e029f6c6a85ee66628ee/pyflakes-2.1.1-py2.py3-none-any.whl (59kB)
    100% |████████████████████████████████| 61kB 125kB/s 
Collecting pycodestyle<2.6.0,>=2.5.0 (from flake8->-r hax/requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/0e/0c/04a353e104d2f324f8ee5f4b32012618c1c86dd79e52a433b64fceed511b/pycodestyle-2.5.0-py2.py3-none-any.whl (51kB)
    100% |████████████████████████████████| 51kB 496kB/s 
Collecting mccabe<0.7.0,>=0.6.0 (from flake8->-r hax/requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/87/89/479dc97e18549e21354893e4ee4ef36db1d237534982482c3681ee6e7b57/mccabe-0.6.1-py2.py3-none-any.whl
Collecting typed-ast<1.5.0,>=1.4.0 (from mypy->-r hax/requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/34/de/d0cfe2ea7ddfd8b2b8374ed2e04eeb08b6ee6e1e84081d151341bba596e5/typed_ast-1.4.0.tar.gz (206kB)
    100% |████████████████████████████████| 215kB 1.6MB/s 
Collecting typing-extensions>=3.7.4 (from mypy->-r hax/requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/03/92/705fe8aca27678e01bbdd7738173b8e7df0088a2202c80352f664630d638/typing_extensions-3.7.4.1-py3-none-any.whl
Collecting mypy-extensions<0.5.0,>=0.4.0 (from mypy->-r hax/requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/5c/eb/975c7c080f3223a5cdaff09612f3a5221e4ba534f7039db34c35d95fa6a5/mypy_extensions-0.4.3-py2.py3-none-any.whl
Collecting six>=1.4 (from python-consul->-r hax/requirements.txt (line 4))
  Downloading https://files.pythonhosted.org/packages/65/26/32b8464df2a97e6dd1b656ed26b2c194606c16fe163c695a992b36c11cdf/six-1.13.0-py2.py3-none-any.whl
Collecting requests>=2.0 (from python-consul->-r hax/requirements.txt (line 4))
  Using cached https://files.pythonhosted.org/packages/7d/e3/20f3d364d6c8e5d2353c72a67778eb189176f08e873c9900e10c0287b84b/requests-2.21.0-py2.py3-none-any.whl
Collecting chardet<3.1.0,>=3.0.2 (from requests>=2.0->python-consul->-r hax/requirements.txt (line 4))
  Using cached https://files.pythonhosted.org/packages/bc/a9/01ffebfb562e4274b6487b4bb1ddec7ca55ec7510b22e4c51f14098443b8/chardet-3.0.4-py2.py3-none-any.whl
Collecting idna<2.9,>=2.5 (from requests>=2.0->python-consul->-r hax/requirements.txt (line 4))
  Using cached https://files.pythonhosted.org/packages/14/2c/cd551d81dbe15200be1cf41cd03869a46fe7226e7450af7a6545bfc474c9/idna-2.8-py2.py3-none-any.whl
Collecting urllib3<1.25,>=1.21.1 (from requests>=2.0->python-consul->-r hax/requirements.txt (line 4))
  Downloading https://files.pythonhosted.org/packages/01/11/525b02e4acc0c747de8b6ccdab376331597c569c42ea66ab0a1dbd36eca2/urllib3-1.24.3-py2.py3-none-any.whl (118kB)
    100% |████████████████████████████████| 122kB 962kB/s 
Collecting certifi>=2017.4.17 (from requests>=2.0->python-consul->-r hax/requirements.txt (line 4))
  Downloading https://files.pythonhosted.org/packages/18/b0/8146a4f8dd402f60744fa380bc73ca47303cccf8b9190fd16a827281eac2/certifi-2019.9.11-py2.py3-none-any.whl (154kB)
    100% |████████████████████████████████| 163kB 1.7MB/s 
Installing collected packages: typing, entrypoints, pyflakes, pycodestyle, mccabe, flake8, typed-ast, typing-extensions, mypy-extensions, mypy, pkgconfig, six, chardet, idna, urllib3, certifi, requests, python-consul, wheel
  Running setup.py install for typed-ast ... done
Successfully installed certifi-2019.9.11 chardet-3.0.4 entrypoints-0.3 flake8-3.7.9 idna-2.8 mccabe-0.6.1 mypy-0.720 mypy-extensions-0.4.3 pkgconfig-1.5.1 pycodestyle-2.5.0 pyflakes-2.1.1 python-consul-1.1.0 requests-2.21.0 six-1.13.0 typed-ast-1.4.0 typing-3.7.4.1 typing-extensions-3.7.4.1 urllib3-1.24.3 wheel-0.33.6
You are using pip version 9.0.1, however version 19.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
--> Installing cfgen dependencies
Collecting PyYAML (from -r cfgen/requirements.txt (line 1))
  Using cached https://files.pythonhosted.org/packages/e3/e8/b3212641ee2718d556df0f23f78de8303f068fe29cdaa7a91018849582fe/PyYAML-5.1.2.tar.gz
Collecting ply (from -r cfgen/requirements.txt (line 2))
  Using cached https://files.pythonhosted.org/packages/a3/58/35da89ee790598a0700ea49b2a66594140f44dec458c07e8e3d4979137fc/ply-3.11-py2.py3-none-any.whl
Building wheels for collected packages: PyYAML
  Running setup.py bdist_wheel for PyYAML ... done
  Stored in directory: /root/.cache/pip/wheels/d9/45/dd/65f0b38450c47cf7e5312883deb97d065e030c5cca0a365030
Successfully built PyYAML
Installing collected packages: PyYAML, ply
Successfully installed PyYAML-5.1.2 ply-3.11
You are using pip version 9.0.1, however version 19.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
--> Installing cfgen/dhall configs
install: creating directory ‘/opt/seagate/hare/share’
install: creating directory ‘/opt/seagate/hare/share/cfgen’
    linking cfgen/dhall -> /opt/seagate/hare/share/cfgen/dhall
    linking cfgen/examples -> /opt/seagate/hare/share/cfgen/examples
--> Building hax .whl package
  File "setup.py", line 16
    user_options: List[Tuple[str, str, str]] = []
                ^
SyntaxError: invalid syntax
make: *** [hax/dist/hax-0.1.0-cp36-cp36m-linux_x86_64.whl] Error 1
```

</details>

\
[m0reportbug-data.tar.xz](/uploads/5bef093928090c0d90086e21c0d76d88/m0reportbug-data.tar.xz)

### Solution

1. Find what's wrong with Maxim's installation or/and Hare documentation.
2. Update `README_developers.md`.

--- a/README_developers.md
+++ b/README_developers.md
@@ -8,14 +8,14 @@ The scripts in this repository constitute a middleware layer between [Consul](ht
 ## Dependencies
 
 * Python &geq; 3.6
-* [Consul](https://www.consul.io/downloads.html)
-* [Dhall](https://github.com/dhall-lang/dhall-lang/wiki/Getting-started%3A-Generate-JSON-or-YAML#linux)
 
 ## Installation
 
-* Copy `consul` executable to a `$PATH` directory (e.g.,
-  `/usr/local/bin/`) on each node of the cluster.
-
+* Ensure that `python3` is an alias to Python 3.6 or newer:
+  ```sh
+  $ python3 --version
+  Python 3.6.3
+  ```
 * Ensure that Mero sources are built.
   ```sh
   $M0_SRC_DIR/scripts/m0 make
@@ -26,7 +26,7 @@ The scripts in this repository constitute a middleware layer between [Consul](ht
 
 1. Prepare the node:
    ```sh
-   git clone ssh://git@gitlab.mero.colo.seagate.com:6022/mero/hare.git
+   git clone ssh://git@gitlab.mero.colo.seagate.com:6022/mero/hare.git --recursive
    cd hare
    sudo make devinstall
    ```

